xclim/testing/tests/test_utils.py

```diff
@@ -90,3 +95,54 @@ def test_ensure_chunk_size():
     assert out.chunks[0] == (3, 3, 3, 3, 3, 5)
     assert out.chunks[1] == (10, 11)
     assert out.chunks[2] == (20,)
+
+
+class Test_nan_calc_percentiles:
+    def test_calc_perc_type7(self):
+        # Exemple array from: https://en.wikipedia.org/wiki/Percentile#The_nearest-rank_method
+        arr = np.asarray([15.0, 20.0, 35.0, 40.0, 50.0])
+        res = nan_calc_percentiles(arr, percentiles=[40.0])
```

Comment:
```suggestion
        res = nan_calc_percentiles(arr, percentiles=[40.0], alpha=1, beta=1)
```
I know these are the default values, but just for clarity.